README.md

```diff
@@ -155,6 +155,7 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/deno_websit
 - [Bartek Iwańczuk - Deno internals, how modern runtime is built - Deno Warsaw](https://www.youtube.com/watch?v=qt7fbmypAFk)
     - [Slides](https://docs.google.com/presentation/d/1LYNGpyjx9PemL-P__7hVC8mSqkX-jL8VQLMhCRehy00/edit?usp=sharing)
 - [Ryan Dahl & Kitson Kelly: Deno is a New Way to JavaScript - TSConf 2019](https://www.youtube.com/watch?v=1gIiZfSbEAE)
+- [dotJS 2019 - Bert Belder - Deno](https://www.youtube.com/watch?v=puXyo1jGQys)
```

Comment:
```suggestion
- [Bert Belder - Deno - dotJS 2019](https://www.youtube.com/watch?v=puXyo1jGQys)
```
Could you insert it a line before to keep alphabetical order?

Reply:
Sure, will do